timemachine/potentials/bonded.py

```diff
@@ -1,18 +1,26 @@
 import jax.numpy as np
 
-def centroid_restraint(conf, params, box, lamb, masses, group_a_idxs, group_b_idxs, kb, b0):
```

Comment:
Nice! Making the centroid definitions independent of the system masses is a good idea, I think.

Not necessarily something to adjust in the current PR, but rather than depending on physical `masses`, what would you think about having this depend on a normalized `weights` parameter?

Reply:
definitely doable, we just need to also support the same behavior in the RMSD restraint.